openvpn/protonvpn/ch.protonvpn.com.tcp.ovpn

```diff
@@ -48,7 +48,7 @@ persist-tun
 reneg-sec 0
 
 remote-cert-tls server
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass
```

Comment:
why are you removing the credentials file? This is created by the container when run to provide the details..

Reply:
if you can explain why this was removed, then we cna look at merging this

Reply:
@casaper can you please comment?

Reply:
oh

@pkishino  Sorry, I kind of lost track here. 

No, that change was accidental. Must have been a glitch when I reverted these changes and this was probably forgotten. 

So you're perfectly right. This change mustn't be merged, because it would render the config broken if used.

I'm sorry, I kinda forgot about having placed this PR  here.

I do not insist in keeping it.

Someone stated the config also works fine for her/him even with servers not deemed p2p eligible by protonvpns page, so I guess we can safely drop my PR here.

Sorry to have kept you waiting.